solana/modules/token_bridge/client/src/main.rs

```diff
@@ -70,6 +63,10 @@ struct Config {
 type Error = Box<dyn std::error::Error>;
 type CommmandResult = Result<Option<Transaction>, Error>;
 
+// [`get_recent_blockhash`] is deprecated, but devnet deployment hangs using the
```

Comment:
Might be worth rewording the comment to suggest it only _may_ hang in devnet as the coode actually does have the updated method below.